scripts/album.js

```diff
@@ -191,12 +193,13 @@ var setupSeekBars = function() {
 };
 
```

Comment:
I'm having trouble following the feedback for this. I haven't gotten to the filterTimeCode portion of the assignment yet. This commit is only the first 2 functions of the assignment - setCurrentTimeInPlayerBar() and updateSeekBarWhileSongPlays()